js/glowingbear.js

```diff
@@ -746,6 +746,31 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
         }
     };
 
+    function nickFromBufferline(bufferline) {
```

Comment:
I didn't duplicate anything that I know of. Maybe this function belongs in models.js but it doesn't duplicate functionality from there.